src/db/repository/postgres/WishCardsRepository.ts

```diff
@@ -0,0 +1,67 @@
+import { Kysely, UpdateObject } from 'kysely';
+
+import { DB, Wishcards } from '../../types/generated/database';
+
+export type WishcardsUpdateParams = Omit<
+	UpdateObject<DB, 'wishcards'>,
+	'id' | 'created_at' | 'updated_at' | 'agency_id' | 'child_id'
+>;
+export type WishcardsCreateParams = Omit<
+	Wishcards,
+	'id' | 'created_at' | 'updated_at' | 'deleted_at' | 'status'
+>;
+
+export default class WishcardsRepository {
+	constructor(private readonly database: Kysely<DB>) {}
+
+	create(createParams: WishcardsCreateParams) {
+		return this.database
+			.insertInto('wishcards')
+			.values(createParams)
+			.returningAll()
+			.executeTakeFirstOrThrow();
+	}
+
+	getAll() {
+		return this.database.selectFrom('wishcards').selectAll().execute();
+	}
+
+	getByWishItemName(itemName: string) {
+		return this.database
+			.selectFrom('wishcards')
+			.innerJoin('items', 'wishcards.item_id', 'items.id')
```

Comment:
```suggestion
			.innerJoin('items', 'items.id', 'wishcards.item_id')
```

fixed the join params